### PR TITLE
Update upcoming-breaking-changes.md

### PR DESCRIPTION
**Pull request by Nickcandy**
# PR Summary

Update upcoming-breaking-changes.md

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: docs-conceptual/azps-9.2.0/upcoming-breaking-changes.md [hunk 1]
```diff
@@ -115,18 +115,18 @@ title: Upcoming breaking changes in Azure PowerShell
 ### `New-AzDisk`
 
 - Cmdlet breaking-change will happen to all parameter set
-  - Starting in May 2023 the "New-AzDisk" cmdlet will deploy with the Trusted Launch configuration by default. This includes defaulting the "HyperVGeneration" parameter to "v2". To know more about Trusted Launch, please visit [https://learn.microsoft.com/azure/virtual-machines/trusted-launch](/azure/virtual-machines/trusted-launch)
+  - Starting in May 2023 the "New-AzDisk" cmdlet will deploy with the Trusted Launch configuration by default. This includes defaulting the "HyperVGeneration" parameter to "v2". To know more about Trusted Launch, please visit https://docs.microsoft.com/en-us/azure/virtual-machines/trusted-launch
```
Comment (mikefrobbins) [inline docs-conceptual/azps-9.2.0/upcoming-breaking-changes.md:118]:
```suggestion
  - Starting in May 2023 the "New-AzDisk" cmdlet will deploy with the Trusted Launch configuration by default. This includes defaulting the "HyperVGeneration" parameter to "v2". To know more about Trusted Launch, please visit [https://learn.microsoft.com/azure/virtual-machines/trusted-launch](/azure/virtual-machines/trusted-launch).
```

File: docs-conceptual/azps-9.2.0/upcoming-breaking-changes.md [hunk 2]
```diff
@@ -115,18 +115,18 @@ title: Upcoming breaking changes in Azure PowerShell
...
 
 ### `New-AzVM`
 
 - Cmdlet breaking-change will happen to all parameter set
-  - Starting in May 2023 the "New-AzVM" cmdlet will deploy with the Trusted Launch configuration by default. To know more about Trusted Launch, please visit [https://learn.microsoft.com/azure/virtual-machines/trusted-launch](/azure/virtual-machines/trusted-launch)
+  - Starting in May 2023 the "New-AzVM" cmdlet will deploy with the Trusted Launch configuration by default. To know more about Trusted Launch, please visit https://docs.microsoft.com/en-us/azure/virtual-machines/trusted-launch
```
Comment (mikefrobbins) [inline docs-conceptual/azps-9.2.0/upcoming-breaking-changes.md:123]:
```suggestion
  - Starting in May 2023 the "New-AzVM" cmdlet will deploy with the Trusted Launch configuration by default. To know more about Trusted Launch, please visit [https://learn.microsoft.com/azure/virtual-machines/trusted-launch](/azure/virtual-machines/trusted-launch).
```

File: docs-conceptual/azps-9.2.0/upcoming-breaking-changes.md [hunk 3]
```diff
@@ -115,18 +115,18 @@ title: Upcoming breaking changes in Azure PowerShell
...
   - It is recommended to use parameter "-PublicIpSku Standard" in order to create a new VM with a Standard public IP.Specifying zone(s) using the "-Zone" parameter will also result in a Standard public IP.If "-Zone" and "-PublicIpSku" are not specified, the VM will be created with a Basic public IP instead.Please note that the Standard SKU IPs will become the default behavior for VM creation in the future
 
 ### `New-AzVmss`
 
 - Cmdlet breaking-change will happen to all parameter set
-  - Starting in May 2023 the "New-AzVmss" cmdlet will deploy with the Trusted Launch configuration by default. To know more about Trusted Launch, please visit [https://learn.microsoft.com/azure/virtual-machines/trusted-launch](/azure/virtual-machines/trusted-launch)
+  - Starting in May 2023 the "New-AzVmss" cmdlet will deploy with the Trusted Launch configuration by default. To know more about Trusted Launch, please visit https://docs.microsoft.com/en-us/azure/virtual-machines/trusted-launch
```
Comment (mikefrobbins) [inline docs-conceptual/azps-9.2.0/upcoming-breaking-changes.md:129]:
```suggestion
  - Starting in May 2023 the "New-AzVmss" cmdlet will deploy with the Trusted Launch configuration by default. To know more about Trusted Launch, please visit [https://learn.microsoft.com/azure/virtual-machines/trusted-launch](/azure/virtual-machines/trusted-launch).
```

**Review by mikefrobbins — CHANGES_REQUESTED** [review]
A few links are generating docs build system warnings.
- Bare URL's should be replaced by links that don't include the culture so the customer lands on a page using their own culture. 
- Relative URL's should be used in the docs system so they work in air-gapped environments.

Please accept the suggestions I've added.

**Comment by Nickcandy** [comment]
@mikefrobbins Suggestions accepted. Thanks Mike.

**Review by mikefrobbins — APPROVED** [review]
LGTM

**Pull request merged**